selfdrive/car/subaru/values.py

```diff
@@ -89,10 +97,14 @@ def init_make(self, CP: car.CarParams):
       self.footnotes.append(Footnote.EXP_LONG)
 
 
-@dataclass(frozen=True)
+@dataclass
 class SubaruPlatformConfig(PlatformConfig):
   dbc_dict: DbcDict = field(default_factory=lambda: dbc_dict('subaru_global_2017_generated', None))
 
+  def init(self):
```

Comment:
@sshane this should help with some of the toyota copy/paste, we can set the DBC based on a flag ex: nodsu

Reply:
Nice, if there's going to be less grouping, I wanted a way to easily show exceptions like this where one or two platforms may differ from the rest.